calibration/CTDMOG/CGINS-CTDMOG-13432__20161019.csv

```diff
@@ -17,9 +17,9 @@ serial,name,value,notes
 37-13432,CC_ptca1,8.147910e+000,
 37-13432,CC_ptca2,-2.533877e-001,
 37-13432,CC_ptcb0,2.502625e+001,
-37-13432,CC_ptcb1,4.987531e-005,
+37-13432,CC_ptcb1,4.987532e-05,
 37-13432,CC_ptcb2,0.00E+00,
 37-13432,CC_ptempa0,1.504269e+002,
-37-13432,CC_ptempa1,-6.208685e-002,
+37-13432,CC_ptempa1,-6.208686e-02,
 37-13432,CC_ptempa2,-9.951629e-007,
-37-13432,CC_wbotc,2.5058e-007,
+37-13432,CC_wbotc,2.505781e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT